demisto_sdk/commands/secrets/tests/secrets_test.py

```diff
@@ -313,3 +313,31 @@ def test_get_all_diff_text_files(self, mocker):
         validator.prev_ver = 'Testing_branch'
         assert validator.get_all_diff_text_files('master', True) == ['Packs/Integrations/integration/testing.py']
         assert validator.get_all_diff_text_files('master', False) == ['Packs/Integrations/integration/testing.py']
+
+    def test_remove_secrets_disabled_line(self):
+        """
+        Given
+            1. String with a line that are disable-secrets-detection
+            1. String with a lines whit "disable-secrets-detection-start" & "disable-secrets-detection-end"
+        When
+            Removing the lines from the file
+        Then
+            Ensure secrets that that are in this lines isn't in the output.
+        """
+        file_contents = '''
+        import
+        8.8.8.8 # disable-secrets-detection
+        end
+        '''
+        file_contents = self.validator.remove_secrets_disabled_line(file_contents)
+        assert "8.8.8.8" not in file_contents
+
+        file_contents1 = '''
+        import
+        8.8.8.8 # disable-secrets-detection-start
+        4.4.4.4
+        end # disable-secrets-detection-end
+        '''
```

Comment:
Is this the same behavior as was today? even if the comment is after the secret (like 8.8.8.8 it is being ignored?)

Reply:
yes

Reply:
Let's add another secret here and let's make sure it is picked up